client/.env

```diff
@@ -1,2 +0,0 @@
-REACT_APP_AUTH0_DOMAIN = dangkimberly26.us.auth0.com
-REACT_APP_AUTH0_CLIENT_ID= CTG6Yr3WKq5bXwS4K47aWrqUO7KpabAh
```

Comment:
If these really were secure keys deleting them in a commit isn't sufficient because they are still in the Git history and can be stolen.

Since these are not actually sensitive (they cannot be secret because they are in the browser of any user of the app) I think it's fine but just in case it comes up again, if you have time read over GitHub's explanation of what to do:

https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository

(If you leaked anything into the world, you would need to change the secret(s) because they are considered compromised)